keyboards/ll3macorn/bongopad/config.h

```diff
@@ -0,0 +1,42 @@
+// Copyright 2021 Ll3macorn (@ll3macorn)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#pragma once
+#define CONFIG_H
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x1234
```

Comment:
This vendor ID is in use, and needs to be changed. 

https://www.the-sz.com/products/usbid/index.php?v=0x1234&p=&n=